recipes/poco/all/conanfile.py

```diff
@@ -169,52 +176,25 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
         tools.rmdir(os.path.join(self.package_folder, "cmake"))
 
-    def package_info(self):
-        option_libs = OrderedDict((
-            ("enable_zip", "PocoZip"),
-            ("enable_pdf", "PocoPDF"),
-            ("enable_sevenzip", "PocoSevenZip"),
-            ("enable_netssl_win", "PocoNetSSLWin"),
-            ("enable_netssl", "PocoNetSSL"),
-            ("enable_jwt", "PocoJWT"),
-            ("enable_util", "PocoUtil"),
-            ("enable_apacheconnector", "PocoApacheConnector"),
-            ("enable_data_sqlite", "PocoDataSQLite"),
-            ("enable_data_odbc", "PocoDataODBC"),
-            ("enable_data_postgresql", "DataPostgreSQL"),
-            ("enable_data_mysql", "PocoDataMySQL"),
-            ("enable_redis", "PocoRedis"),
-            ("enable_mongodb", "PocoMongoDB"),
-            ("enable_net", "PocoNet"),
-            ("enable_encodings", "PocoEncodings"),
-            ("enable_xml", "PocoXML"),
-            ("enable_data", "PocoData"),
-            ("enable_crypto", "PocoCrypto"),
-            ("enable_json", "PocoJSON"),
-        ))
-
-        if tools.Version(self.version) < "1.9":
-            option_libs.pop("enable_encodings")
-        if tools.Version(self.version) < "1.10":
-            option_libs.pop("enable_jwt")
-            option_libs.pop("enable_data_postgresql")
+    @property
+    def _ordered_libs(self):
+        libs = []
+        for compname in self._poco_ordered_components:
+            comp_options = self._poco_component_tree[compname]
+            if comp_options.is_lib:
+                if not comp_options.option:
+                    libs.append(compname)
+                elif self.options.get_safe(comp_options.option, False):
+                    libs.append(compname)
+        return libs
 
+    def package_info(self):
         suffix = str(self.settings.compiler.runtime).lower()  \
                  if self.settings.compiler == "Visual Studio" and not self.options.shared \
                  else ("d" if self.settings.build_type == "Debug" else "")
-        for option, lib in option_libs.items():
-            if self.options.get_safe(option, False):
-                if self.settings.os == "Windows" and option == "enable_netssl" and self.options.enable_netssl_win:
-                    continue
-
-                if self.settings.os != "Windows" and option == "enable_netssl_win":
-                    continue
-
-                self.cpp_info.libs.append("%s%s" % (lib, suffix))
-
-        self.cpp_info.libs.append("PocoFoundation%s" % suffix)
 
-        # in linux we need to link also with these libs
+        self.cpp_info.libs = list("{}{}".format(lib, suffix) for lib in self._ordered_libs)
```

Comment:
I expected to see components here, I missing something?

Reply:
That was not the goal of this pr.
This pr explicitly sets `self._cmake.definitions["POCO_UNBUNDLED"] = True`.
Before, some things could potentially be built from bundled sources.

Reply:
I propose to finish the boost components pr first as a poc before adding components to this one using a similar approach.

Reply:
sounds good!